packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -149,16 +163,31 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
         resultHintString = FaceMatchStateStringMap[faceMatchState];
       }
 
+      // If the face is outside the oval set the aria-label to a string about centering face in oval
+      let a11yHintString = resultHintString;
+      if (faceMatchState === FaceMatchState.OFF_CENTER) {
+        a11yHintString = FaceMatchStateStringMap[faceMatchState];
+      }
+
+      // If the face match percentage reaches 50% append it to the a11y label
+      if (
+        faceMatchState === FaceMatchState.TOO_FAR &&
+        faceMatchPercentage! > 50
+      ) {
+        a11yHintString = hintDisplayText.hintMatchIndicatorText;
+      }
```

Comment:
Is it possible for both these cases to be truthy?

Reply:
Don't think so at this time since those are distinctly different faceMatchStates

Reply:
Cool, that's how i was reading it. Not a blocker but might make sense to put this in a util and/or update to an `if`/`else`